types/aws-lambda/trigger/codebuild-cloudwatch.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Handler } from '../handler';
+import { CodeBuildCloudWatchStateEvent } from './codebuild-cloudwatch-state';
+
+export type CodeBuildCloudWatchHandler = Handler<CodeBuildCloudWatchEvent, void>;
+
+export type CodeBuildCloudWatchEvent = CodeBuildCloudWatchStateEvent;
```

Comment:
Not sure what these are for? I don't think there are any straight up alias types that are not for compatibility, but there could be some specific reason here.

Reply:
idea behind is that there can also be another type of `CodeBuildCloudWatchEvent` - for example `CodeBuildCloudWatchPhaseEvent`
It's similar to 
```
export type CodePipelineCloudWatchEvent =
    | CodePipelineCloudWatchPipelineEvent
    | CodePipelineCloudWatchStageEvent
    | CodePipelineCloudWatchActionEvent;

```

Reply:
That union was added with all entries at the same time (in #31311), presumably because it's quite likely you would send all three event types to the same handler. Otherwise it's not really something this package does, and as is, it's not safe to add more alternatives as it's not currently a union so it doesn't have a discriminant to be tested. For example:

```ts
async function handler(event: CodeBuildCloudWatchEvent) {
    event["build-id"] // works now, but may not if other alternatives were added
}
```

It would be simple enough to add this later once there's a clear case for it, so I would remove this for now.